ota-plus-web/app/com/advancedtelematic/controllers/ClientToolController.scala

```diff
@@ -156,7 +156,8 @@ class ClientToolController @Inject()(
         }
       }
       val authParams = if(conf.getOptional[String]("authplus.host").isDefined) {
-        await(oidcGateway.getUserInfo(accessToken).flatMap(ic => getOAuth2Params(namespace, ic.userId)))
+        val userId = UserId("the-value-of-this-userId-is-not-check-by-authplus-but-it-should-be")
```

Comment:
Let's not be this explicit. It will be the `subject` of the token issued to the cli user.

Reply:
Alright 😄 I updated to using an empty string and adding a 'todo'.